src/Resources/Locales/en.json

```diff
@@ -1124,5 +1118,23 @@
         "days": "days",
         "year": "year",
         "years": "years"
+    },
+    "resourcesPicker": {
+        "select": "Select a resource",
+        "noOption": "No resources found",
+        "loadingResources": "Loading resources",
```

Comment:
please name the resource keys for what they are used for and not for the value they represent. `enterEnvironmentUrl` --> `environmentDropdownPlaceholder`. When we change the text it no longer makes sense. Ex: `select` --> `storageContainerPlaceholderText`

Reply:
changed: https://github.com/microsoft/iot-cardboard-js/pull/684/commits/33de50355aab694cfa0eb40d584c3eaa40f80683#diff-f9ebb0fc4e7c56a13821e99a435e6d222ed423f8aced31d6946e848882c9dc72